mtgjson4/__init__.py

```diff
@@ -53,12 +53,34 @@
 COMPILED_LIST_OUTPUT: str = "CompiledList"
 DECK_LISTS_OUTPUT: str = "DeckLists"
 KEY_WORDS_OUTPUT: str = "Keywords"
-MODERN_OUTPUT: str = "Modern"
 REFERRAL_DB_OUTPUT: str = "ReferralMap"
 SET_LIST_OUTPUT: str = "SetList"
-STANDARD_OUTPUT: str = "Standard"
 VERSION_OUTPUT: str = "version"
+
+STANDARD_OUTPUT: str = "Standard"
+MODERN_OUTPUT: str = "Modern"
 VINTAGE_OUTPUT: str = "Vintage"
+LEGACY_OUTPUT: str = "Legacy"
```

Comment:
I do want to make a `globals` class in the near future since this is getting a bit large for our init